### PR TITLE
Update README.md and upload blockdiagram.jpg

### PR DESCRIPTION
**Pull request by Wesley-Du**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,459 +2,49 @@
...
+   <p align="center"> <img src="docs/docs/Image/blockdiagram.jpg" height=240px； weight=718px；"><br></p>
+</div>
 
-**By default, you can build a Analytics-Zoo:default image with latest nightly-build Analytics-Zoo distributions:**
 
-```bash
-sudo docker build --rm -t intelanalytics/analytics-zoo:default .
-```
+- **Integrated Analytics and AI Pipelines** for easily prototyping and deploying end-to-end AI applications. 
+  - Write [TensorFlow](https://analytics-zoo.github.io/master/##distributed-tensorflow-and-keras-on-sparkbigdl) or [PyTorch](https://analytics-zoo.github.io/master/#ProgrammingGuide/pytorch/) inline with Spark code for distributed training and inference.
```
Comment (jason-dai) [inline README.md]: Use this link for TF: https://analytics-zoo.github.io/master/#ProgrammingGuide/tensorflow/#tfpark-api

File: README.md [hunk 2]
```diff
@@ -2,459 +2,49 @@
...
+  - Native deep learning (TensorFlow/Keras/PyTorch/BigDL) support in [Spark ML](https://analytics-zoo.github.io/master/##nnframes) Pipelines.
```
Comment (jason-dai) [inline README.md]: Use this link instead

File: README.md [hunk 3]
```diff
@@ -2,459 +2,49 @@
...
+  - Directly run Ray programs on big data cluster through [RayOnSpark](https://analytics-zoo.github.io/master/#ProgrammingGuide/rayonspark/). 
+  - Plain Java/Python APIs for (TensorFlow/PyTorch/BigDL/OpenVINO) [Model Inference](https://analytics-zoo.github.io/master/##model-serving).
```
Comment (jason-dai) [inline README.md]: https://analytics-zoo.github.io/master/#ProgrammingGuide/inference/

File: README.md [hunk 4]
```diff
@@ -2,459 +2,49 @@
...
-**If you need http and https proxy to build the image:**
-```bash
-sudo docker build \
-    --build-arg http_proxy=http://your-proxy-host:your-proxy-port \
-    --build-arg https_proxy=https://your-proxy-host:your-proxy-port \
-    --rm -t intelanalytics/analytics-zoo:default .
-```
+- High-Level **ML Workflow** that automates the process of building large-scale machine learning applications.
+  - Automatically distributed real-time serving (for TensorFlow/PyTorch/Caffe/BigDL/OpenVINO models) with a simple pub/sub API. 
+  - Scalable AutoML for time series prediction (that automatically generates features, selects models and tunes hyperparameters).
```
Comment (jason-dai) [inline README.md]: https://github.com/intel-analytics/analytics-zoo/blob/automl/pyzoo/zoo/automl/README.md

File: README.md [hunk 5]
```diff
@@ -2,459 +2,49 @@
...
 
-### _Notice_
-**If you need nightly build version of Analytics-Zoo, please pull the image form Dockerhub with:**
-```bash
-sudo docker pull intelanalytics/analytics-zoo:latest
-```
+# <font size="6">How to use Analytics Zoo? </font>
 
-**Please follow the readme in each app folder to test the jupyter notebooks !!!**
+- Quick start with Analytics Zoo using the pre-built [Docker Image](https://analytics-zoo.github.io/master/##docker-images-and-builders).
```
Comment (jason-dai) [inline README.md]: you need to put this into another page under user guide

**Comment by Wesley-Du** [comment]
close

**Pull request closed without merging**